### PR TITLE
Feature/adapter improvements

### PR DESCRIPTION
**Pull request by gaya3-vijayakumar**
## What

- Change Mistral model to a free text field in UI
- Change Milvus implementation to push the credentials
- Change Milvus UI to taken token optionally for local server
- Updated vecs version for Supabase

## Why

- Milvus test connection is failing. Hence fixes required
- Mistral needs to support a new model to test efficiency
- Supabase uses older version of vecs which makes API calls to add docs to fail. 

## How

-

## Relevant Docs

-

## Related Issues or PRs

- https://zipstack.atlassian.net/browse/UN-872
- https://zipstack.atlassian.net/browse/UN-845

## Dependencies Versions / Env Variables

-

## Notes on Testing

...

## Screenshots
image

image

## Checklist

I have read and understood the [Contribution Guidelines]().

### CODE REVIEW

File: src/unstract/adapters/llm/mistral/src/static/json_schema.json [hunk 1]
```diff
@@ -25,8 +14,10 @@
       "description": "Provide a unique name for this adapter instance. Example: mistral-group-1"
     },
     "model": {
+      "type": "string",
       "title": "Model",
-      "$ref": "#/definitions/models"
+      "default": "mistral-large-latest",
+      "description": "Provide the model name to be used. Example: mistral-tiny, mistral-small, mistral-medium, mistral-large-latest"
```
Comment (chandrasekharan-zipstack) [inline src/unstract/adapters/llm/mistral/src/static/json_schema.json:20]: NIT: The examples would help, alternatively we can also link to mistral docs on supported models - but we need to check if its doable with JSON schema and our current implementation

**Review by chandrasekharan-zipstack — COMMENTED** [review]
Are there more changes coming in? Can you bump the adapter version so that it can be published and used

**Review by chandrasekharan-zipstack — APPROVED** [review]
LGTM, but make sure you update the `pdm.lock` file as well

**Comment by gaya3-vijayakumar** [comment]
> LGTM, but make sure you update the `pdm.lock` file as well

There was no pdm.lock commited to github. But I added one now.

**Review by jaseemjaskp — APPROVED** [review]
LGTM

**Pull request merged**